ajaxSample.php

```diff
@@ -0,0 +1,46 @@
+<?php
+//echo '====hello'; 
```

Comment:
File Header is missing, File name is not correct ajaxSample.php.
This is not a filename, it should be some proper name like search.php or searchEmployee.php etc